Up2dateService/Up2dateService/SetupManager/SetupManager.cs

```diff
@@ -264,15 +378,51 @@ private void RefreshPackageList()
                     updatedPackage.InstallDate = null;
                     updatedPackage.EstimatedSize = null;
                     updatedPackage.UrlInfoAbout = null;
-                    if (updatedPackage.Status != PackageStatus.Downloading && updatedPackage.Status != PackageStatus.Installing)
-                    {
+                    if (updatedPackage.Status != PackageStatus.Downloading &&
+                        updatedPackage.Status != PackageStatus.Installing)
                         updatedPackage.Status = PackageStatus.Downloaded;
-                    }
                 }
+
+                if (updatedPackage.Filepath.Contains(NugetExtension))
```

Comment:
I guess this code should only be used if Choco is installed.
If not, we should degrade gracefully ignoring nuget packages.